app/src/main/java/com/movie/app/service/MovieService.java

```diff
@@ -27,8 +34,40 @@ public List<Movie> getMoviesByGenre(String genre) {
     }
 
     @Cacheable(value = "Movies", key = "all", cacheManager = "contentCacheManager")
-    public List<Movie> getMoviesByGenre() {
+    public List<Movie> getMoviesAll() {
         return movieRepository.findAll();
     }
 
+    public Movie postMovie(MovieRequestDto requestDto) {
+        Movie movie = new Movie(requestDto);
+        movieRepository.save(movie);
+        return movie;
+    }
+
+    @Transactional
+    public Movie ticketing(Long id, TicketingRequestDto requestDto) {
+        Movie movie = movieRepository.findById(id).orElseThrow(
+                () -> new NullPointerException("There is no id at DB.")
+        );
+
+        if(movie==null) {
+            return movie;
+        }
```

Comment:
```suggestion
        Movie movie = movieRepository.findById(id).orElseThrow(
                () -> new NullPointerException("There is no id at DB.")
        );
```
movie에 대해 orElseThrow를 하셨다면 movie가 null일 수는 없겠네요!
if 조건문은 없어도 되지 않을까요?ㅎㅎ